packages/swapper/src/swappers/LifiSwapper/endpoints.ts

```diff
@@ -98,7 +100,9 @@ export const lifiApi: SwapperApi = {
     const tradeRateResult = await getTradeRate(input as GetEvmTradeRateInput, deps, lifiChainMap)
 
     return tradeRateResult.map(quote =>
-      quote.map(({ selectedLifiRoute, ...tradeQuote }) => {
+      quote.map(tradeQuote => {
+        const { selectedLifiRoute } = tradeQuote
+
```

Comment:
This is not just stylistic. We do return `selectedLifiRoute` as part of the `TradeRate`, so that when we we hit the case of "unstable route not available anymore the second time around " this PR fixes and use original rate, `getTradeQuote()` doesn't throw with `missing selectedLifiRoute`